README.md

```diff
@@ -84,24 +84,23 @@ For **ARM Devices** (such as Raspberry Pi) things might be a bit different. If y
 1. From that recovery image, you will need the Widevine files located in /opt/google/chrome/libwidevinecdm*.so.
 1. These files need to be copied to the `<kodi-profile>/cdm` folder.
 
-_NOTE: Keep in mind that you might need to try multiple recovery images before you find a working one._ 
+_Note: Keep in mind you might need to try multiple recovery images before you find a working one._ 
 
 # Acknowledgement #
-The first idea for Retrospect/XBMC Online TV/XOT-Uzg came from a script by
-BaKMaN.
+The first idea for Retrospect/XBMC Online TV/XOT-Uzg came from a script by BaKMaN.
 
 # Copyrights and Licenses #
-Retrospect is licensed under a _Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International_, see: [LICENSE.md](./LICENSE.md)
+Retrospect is licensed _Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International_, see: [LICENSE.md](./LICENSE.md)
 
-The official add-on package for Retrospect may not be distributed via other repositories than the official Kodi add-on repository.
+The official add-on package for Retrospect is not distributed via other repositories than the official Kodi add-on repository.
 
 ### Disclaimer ###
-Retrospect or Rieter.net are not connected to or in any other way affiliated with Kodi, Team Kodi or the XBMC Foundation. Furthermore, any software, addons, or products offered by Retrospect or Rieter.net will only receive support in its [Kodi forum](https://forum.kodi.tv/showthread.php?tid=25522) and [Github repository](https://github.com/retrospect-addon/plugin.video.retrospect).
+Retrospect or Rieter.net are not connected to or in any other way affiliated with Kodi, Team Kodi or the XBMC Foundation. Furthermore, any software, addons, or products offered by Retrospect or Rieter.net will only receive support in its [Kodi forum](https://forum.kodi.tv/showthread.php?tid=25522) and [GitHub repository](https://github.com/retrospect-addon/plugin.video.retrospect).
 
 ### Rules & Terms ###
-As more and more people are starting to make channels for Retrospect, we want to lay out some rules and terms for the channels which we will host. Please stick to them before asking us to merge your work with the master branch:
+With more people adding channels for Retrospect, here are some rules and terms for channels we will host. Please ensure your channel qualifies before asking for it to be merged with the master branch:
```

Comment:
Perhaps this should be more something like: "If you want to contribute by adding a new channel, feel free to start a new PR with those changes. Before doing this please ensure the new channel qualifies with the following rules:

Reply:
Sorry for me commenting a little late. 

I think this would be great! To get other people than already overworked you, Bas, to handle some channels would be good.